src/OpenTelemetry/CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
   which could have led to a measurement being dropped.
   ([#5546](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5546))
 
+* **Experimental (pre-release builds only):** Exposed
+  `FixedSizeExemplarReservoir` as a public API to support custom implementations
+  of `ExemplarReservoir` which may be configured using the
+  `ExemplarReservoirFactory` property on the View API.
+  ([#5558](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5558))
```

Comment:
would we be showing a doc with an example reservoir? The PR description has one, but wondering if it is planned for the doc itself?

Reply:
I've been trying to keep #2527 updated with tasks needing to be done. I do have a task on there for docs. I was thinking yes we would include custom reservoir support in there with a note is it experimental.